### DISCUSSION TITLE
exercises for begginers

### DISCUSSION THREAD

**Discussion by ronmikailov** [Cyfrin/foundry-full-course-cu#1924] [General]
hello fellow peogrammers, can you send some good platforms to exercise in coding in solidity? im new to this course and proggraming, and i want to train on things i learn in the course. much appreciated!

**Comment by EngrPips** [comment]
Hello @ronmikailov, It is amazing that you just started learning and are already looking to exercise what you learned. Please keep up the spirit. Honestly, I believe you try building everything that you built in the course while following Patrick alone yourself, and this is the greatest first step to seeing that you truly understand everything you have been learning
